robottelo/cli/factory.py

```diff
@@ -1840,6 +1840,7 @@ def make_compute_resource(options=None):
 
     Options::
 
+        --datacenter DATACENTER       for RHEV, VMware Datacenter
```

Comment:
Update docstring and parameters to actual state:
```
def make_compute_resource(options=None):
    """
    Usage::

        hammer compute-resource create [OPTIONS]

    Options::

        --caching-enabled CACHING_ENABLED    enable caching, for VMware only
                                             One of true/false, yes/no, 1/0.
        --datacenter DATACENTER       for RHEV, VMware Datacenter
        --description DESCRIPTION
        --display-type DISPLAY_TYPE   for Libvirt only
                                      Possible value(s): 'VNC', 'SPICE'
        --location-ids LOCATION_IDS   Comma separated list of values.
        --location-titles LOCATION_TITLES    Comma separated list of values.
        --locations LOCATION_NAMES    Comma separated list of values.
        --name NAME
        --organization-ids ORGANIZATION_IDS  Comma separated list of values.
        --organization-titles ORGANIZATION_TITLES    Comma separated list of
                                                     values.
        --organizations ORGANIZATION_NAMES   Comma separated list of values.
        --password PASSWORD           Password for Ovirt, EC2, Vmware,
                                      Openstack. Secret key for EC2
        --provider PROVIDER           Providers include Libvirt, Ovirt, EC2,
                                      Vmware, Openstack, Rackspace, GCE
        --region REGION               for EC2 only
        --server SERVER               for Vmware
        --set-console-password SET_CONSOLE_PASSWORD for Libvirt and Vmware only
                                                    One of true/false,
                                                    yes/no, 1/0.
        --tenant TENANT               for Openstack only
        --url URL                     URL for Libvirt, Ovirt, and Openstack
        --user USER                   Username for Ovirt, EC2, Vmware,
                                      Openstack. Secret key for EC2
        --uuid UUID                   for Ovirt, Vmware Datacenter
        -h, --help                    print help

    """
    args = {
        u'caching-enabled': None,
        u'datacenter': None,
        u'description': None,
        u'location-ids': None,
        u'location-titles': None,
        u'locations': None,
        u'name': gen_alphanumeric(),
        u'organization-ids': None,
        u'organization-titles': None,
        u'organizations': None,
        u'password': None,
        u'provider': None,
        u'region': None,
        u'server': None,
        u'set-console-password': None,
        u'tenant': None,
        u'url': None,
        u'user': None,
        u'uuid': None,
    }
```

Reply:
2)
I would suggest to remove:
```
    #
    # if options is None:
    #     options = {}

    # if options.get('provider') is None:
    #     options['provider'] = FOREMAN_PROVIDERS['libvirt']
        # if options.get('url') is None:
        #     options['url'] = 'qemu+tcp://localhost:16509/system'
```
with corresponding re-test for all cases involved.

Reply:
@oshtaier, think this is not directly related to the current tests, 
there is too many tests involved in different test modules , can be done in a  separate PR (that  can be also blocked by other reviewer)
of course will update that create usage help doc string